ONSdigital/rm-collection-exercise-service
# Update CodeQL to v2

# What and why?
This PR updates the deprecated CodeQL to v2. CodeQL wasn't actually running in this repo as it wasn't in the workflow folder. It has now been moved and is running. 
# How to test?
Check [output](https://github.com/ONSdigital/rm-collection-exercise-service/pull/332#:~:text=will%20appear%20on-,this%20overview.,-Once%20you%20merge) of the CodeQl on this PR.
# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&projectKey=RAS&view=detail&selectedIssue=RAS-907
